tidb-cloud/setup-azure-self-hosted-kafka-private-link-service.md

```diff
@@ -0,0 +1,544 @@
+
+The mechanism works as follows:
+
+1. The TiDB Cloud virtual network connects to the Kafka virtual network through private endpoints.
+2. Kafka clients need to communicate directly to all Kafka brokers.
+3. Each Kafka broker is mapped to a unique port of endpoints within the TiDB Cloud virtual network.
+4. Leverage the Kafka bootstrap mechanism and Azure resources to achieve the mapping.
+
+The following diagram shows the mechanism. 
+
+![Connect to Azure Self-Hosted Kafka Private Link Service](/media/tidb-cloud/changefeed/connect-to-azure-self-hosted-kafka-privatelink-service.jpeg)
+
+The document provides an example of connecting to a Kafka Private Link service in Azure. While other configurations are possible based on similar port-mapping principles, this document covers the fundamental setup process of the Kafka Private Link service. For production environments, a more resilient Kafka Private Link service with enhanced operational maintainability and observability is recommended.
+
+## Prerequisites
+
+1. Ensure that you have the following authorization to set up a Kafka Private Link service in your own Azure account. 
+
+    - Manage virtual machines
+    - Manage virtual networks
+    - Manage load balancers
+    - Manage private link services
+    - Connect to virtual machines to configure Kafka nodes
+
+2. [Create a TiDB Cloud Dedicated cluster](/tidb-cloud/create-tidb-cluster.md) on Azure if you do not have one.
+
+3. Get the Kafka deployment information from your [TiDB Cloud Dedicated](/tidb-cloud/select-cluster-tier.md#tidb-cloud-dedicated) cluster.
+
+    1. In the [TiDB Cloud console](https://tidbcloud.com), navigate to the [**Clusters**](https://tidbcloud.com/console/clusters) page, and then click the name of your target cluster to go to its overview page.
+    2. In the left navigation pane, click **Changefeed**.
+    3. On the **Changefeed** page, click **Create Changefeed** in the upper-right corner.
+        1. In **Destination**, select **Kafka**.
+        2. In **Connectivity Method**, select **Private Link**.
+    4. Note down the region information and the subscription of the TiDB Cloud Azure account in **Reminders before proceeding**. You will use it to authorize TiDB Cloud to access the Kafka Private Link service.
+    5. Enter a unique **Kafka Advertised Listener Pattern** for your Kafka Private Link service.
+        1. Input a unique random string. It can only include numbers or lowercase letters. You will use it to generate **Kafka Advertised Listener Pattern** later.
+        2. Click **Check usage and generate** to check if the random string is unique and generate **Kafka Advertised Listener Pattern** that will be used to assemble the EXTERNAL advertised listener for Kafka brokers. 
+
+Note down all the deployment information. You need to use it to configure your Kafka Private Link service later.
+
+The following table shows an example of the deployment information.
```

Comment:
preview of this table: https://pr.pingcap-docsite-preview.pages.dev/tidbcloud/setup-azure-self-hosted-kafka-private-link-service/#:~:text=The%20following%20table%20shows%20an%20example%20of%20the%20deployment%20information.